### PR TITLE
Add file parameter examples for Node and Python

### PR DESCRIPTION
**Pull request by colinking**
Adds examples of accessing file parameters from Node.js and Python. I previously added an example for Shell: https://github.com/airplanedev/examples/pull/13

I renamed the Shell example for consistency. I also updated the Node examples to v18.

I also deployed all of the example tasks into stage and prod under an [`Examples`](https://web.airstage.app/folders/fld20220610zn3o7t1xttn?__env=prod) folder. I also configured the git integration to auto-deploy from `main`.

### CODE REVIEW

**Comment by colinking** [comment]
Current dependencies on/for this PR:
* main
  * **PR #14** <a href="https://app.graphite.dev/github/pr/airplanedev/examples/14" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/airplanedev/examples/14?utm_source=stack-comment).

File: python/file-param/main.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import urllib.request
+
+def main(params):
+    file_url = params["example_file"]["url"]
+    print(f"File URL: {file_url}")
+    
+    response = urllib.request.urlopen(file_url)
+    body = response.read().decode("utf-8")
```
Comment (colinking) [inline python/file-param/main.py:8]: @joshma I just used the approach for Python I found on the internet -- lmk if there's a more idiomatic version.

Comment (joshma) [reply python/file-param/main.py:10]: yeahh it's probably more common to just use `requests` https://pypi.org/project/requests/

```
import requests

r = requests.get(url)
r.text # utf-8 decoded 
```

(the docs even mention it lol https://docs.python.org/3/library/urllib.request.html)

**Review by joshma — APPROVED** [review]
approved but assuming you update the python example (and port that to docs)

**Pull request merged**